robot/markers.py

```diff
@@ -117,24 +90,23 @@ def size(self):
     def pixel_corners(self):
         """Pixel co-ordinates of the of the corners of the marker"""
         # TODO define what the order of these corners are
-        return [tuple(x) for x in self._pixel_corners]
+        return [tuple(x) for x in self._raw_data['pixel_corners']]
 
     @property
     def pixel_centre(self):
         """Pixel co-ordinates of the centre of the marker"""
-        return tuple(self._pixel_centre)
+        return tuple(self._raw_data['pixel_centre'])
 
     # Helper functions, Might need to vary these per-game
 
-    def is_wall_marker(self):
+    def is_wall_marker(self) -> bool:
         """ If the marker is a wall marker """
         return self.id in WALL
 
-    def is_token_marker(self):
+    def is_token_marker(self) -> bool:
         """ If the marker is a token marker """
         return self.id in TOKEN
 
     @property
     def cartesian(self):
         raise NotImplementedError("This is not implemented.")
-        return CartCoord()
```

Comment:
Why not convert this to a return type annotation?
Do we need to change this to being marked as an abstract property?